phinx.json

```diff
@@ -0,0 +1,36 @@
+{
+  "paths": {
+    "migrations": "%%PHINX_CONFIG_DIR%%/migrations"
```

Comment:
Rather than creating duplicated configuration files, wouldn't it be better to ship with a PHP file, and then translate it to YML and JSON at run time?